mon-pix/app/templates/components/reset-password-form.hbs

```diff
@@ -4,18 +4,18 @@
     <img class="pix-logo__image" src="{{rootURL}}/images/pix-logo.svg" alt="Pix">
   </a>
 
-  <div class="sign-form__header-container">
-    <h1 class="sign-form-header__title">{{user.fullName}}</h1>
+  <div class="sign-form__header">
+    <h1 class="sign-form-title1">{{user.fullName}}</h1>
```

Comment:
Est-ce que les éléments de styles de textes trouveraient leur place dans un style/globals/_texts.scss ?

Reply:
Même remarque que le commentaire précédent, ce style reste pour le moment plutôt spécifique à la partie sign-in/reset mot de passe, et il faudrait du travail supplémentaire afin de voir si ça ne pourrait pas être mutualisable avec le reste de style de mon-pix.
On laisse comme ça pour le moment, en gardant en tête qu'on pourrait éventuelllement généraliser ça dans le futur.